src/AWS.Deploy.Recipes/RecipeDefinitions/ConsoleAppECSFargateScheduleTask.recipe

```diff
@@ -10,6 +10,17 @@
     "Description": "Deploy a console application as a scheduled task to Amazon Elastic Container Service (Amazon ECS). Recommended for applications that can be deployed as a container image. If your project does not contain a Dockerfile, one will be generated for the project.",
     "TargetService": "Amazon Elastic Container Service",
 
+    "DisplayedResources": [
+        {
```

Comment:
Can we add the CloudWatch Event and call into something like Event Schedule?

Reply:
Added